proposed/http-middleware/middleware-meta.md

```diff
@@ -270,75 +276,37 @@ additional information._
 
 [promises]: https://promisesaplus.com/
 
-### 5.2 Delegate Design
-
-The `DelegateInterface` defines a single method that accepts a request and
-returns a response. The delegate interface must be implemented by any middleware
-dispatcher that uses middleware implementing `MiddlewareInterface`.
-
-#### Why the term "delegate"?
-
-The term "delegate" means something designated to act for or represent another.
-In terms of middleware design, a delegate is called upon by middleware when the
-middleware is unable to process the request itself; the delegate then processes
-the request _for the original middleware_ in order to return a response.
```

Comment:
I'm both pro- and con- this.

The term `$delegate` was originally coined for this particular proposal. If it doesn't make it into the final version, it's just a bit of historical trivia.

That said, documenting the decision making process (e.g., double-pass to single-pass, delegate to handler) may also answer some questions later — and, more importantly, potentially _prevent_ some.

Thoughts, @shadowhand ?